### PR TITLE
[15.0][OU-ADD] website_sale_attribute_filter_price: Price filter added to website_sale

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT37011

ping @pedrobaeza @chienandalu

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Thanks! :+1:

**Review by pedrobaeza — APPROVED** [review]

File: openupgrade_scripts/scripts/website_sale/15.0.1.1/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+def enable_price_filter_view(env):
+    for website in env["website"].search([]):
+        view = env["ir.ui.view"].search(
+            [
+                ("key", "=", "website_sale_attribute_filter_price.pricefilter"),
+                ("website_id", "=", website.id),
+            ]
+        )
+        if view:
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website_sale/15.0.1.1/pre-migration.py:12]: I'm thinking now... shouldn't you remove the old view?

Comment (CarlosRoca13) [reply openupgrade_scripts/scripts/website_sale/15.0.1.1/pre-migration.py:12]: In my tests, when the old module is uninstalled, the old view disappears. But if, to be on the safe side, you want to deactivate it, that's fine for me.

Comment (pedrobaeza) [reply openupgrade_scripts/scripts/website_sale/15.0.1.1/pre-migration.py:12]: But here we are not uninstalling it, but merging it with `website_sale`, so yes, please do another PR unlinking them.

**Pull request merged**